io.github.andrewmkiss.Demeter.yml

```diff
@@ -0,0 +1,188 @@
+  ## https://github.com/flatpak/flatpak-builder-tools/blob/master/cpan/README.md
+  - name: perl
+    disabled: false
+    no-autogen: true
+    config-opts:
+      - '-des'
+      # Build a shared library.
+      - '-Duseshrplib'
+    build-options:
+      cflags: '-fPIC'
+      ldflags: '-fpic'
+    sources:
+      - type: archive
+        url: https://www.cpan.org/src/5.0/perl-5.36.1.tar.gz
+        sha256: 68203665d8ece02988fc77dc92fccbb297a83a4bb4b8d07558442f978da54cc1
+      - type: shell
+        commands:
+          # Have Flatpak run the GNU-compatible configure script.
+          - 'ln -s configure{.gnu,}'
+    # Restore write permission to the Perl libraries.
+    post-install:
+      - 'chmod -R u+w /app/lib/perl5'
+    # Clean up a bunch of stuff we don't need. Depending on your application,
+    # you may have to drop some of these (e.g. *.pod).
+    cleanup:
+      - '/bin/corelist'
+      - '/bin/cpan'
+      - '/bin/enc2xs'
+      - '/bin/encguess'
+      - '/bin/h2ph'
+      - '/bin/h2xs'
+      - '/bin/instmodsh'
+      - '/bin/json_pp'
+      - '/bin/libnetcfg'
+      - '/bin/perlbug'
+      - '/bin/perldoc'
+      - '/bin/perlivp'
+      - '/bin/perlthanks'
+      - '/bin/piconv'
+      - '/bin/pl2pm'
+      - '/bin/pod*'
+      - '/bin/prove'
+      - '/bin/ptar*'
+      - '/bin/shasum'
+      - '/bin/splain'
+      - '/bin/xsubpp'
+      - '/bin/zipdetails'
+      - '/include'
+      - '/man'
+      - '*.pod'
+
+  ## Using the dependencies from flatpak-cpan-generator
+  # Installs the modules generated by flatpak-cpan-generator.
+  # Note that *any* Makefile.PL-style modules MUST be installed in this one step,
+  # as once perllocal.pod is written in one module, it cannot be modified by others.
+  - name: perl-libs
+    disabled: false
+    buildsystem: simple
+    build-commands:
+      - 'perl-libs/install.sh'
+    # Same as with the Perl module, we need to restore write permission.
+    # However, -f is now passed to avoid errors from trying to touch files from the
+    # above module that are now marked as read-only.
+    post-install:
+      - 'chmod -Rf u+w /app/lib/perl5/site_perl'
+      - 'chmod u+w /app/bin/pdl*'
+    sources:
+      - perl-libs.yml
+    # This step should be customized based on the CPAN packages you're using.
+    cleanup:
+      - '/bin'
+      - '/man'
+
+  ## Demeter
+  - name: demeter
+    disabled: false
+    buildsystem: simple
+    build-options:
+      env:
+        - PERL5LIB=/run/build/demeter:/app/lib/perl5/site_perl
+        - PERLLIB=/app/lib/perl5/site_perl/5.36.1
+    build-commands:
+      - perl Build.PL
+      - sed -i "1653,+3s/25/35/" lib/Demeter/UI/Athena.pm
+      - ./Build
+      - ./Build install
+      - install -Dm 644 metainfo.xml /app/share/metainfo/io.github.andrewmkiss.Demeter.metainfo.xml
+      - install -Dm 644 /app/lib/perl5/site_perl/5.36.1/x86_64-linux/Demeter/share/Demeter_icon.png /app/share/icons/hicolor/48x48/apps/io.github.andrewmkiss.Demeter.png
```

Comment:
Use `perl -e 'use Config; print $Config{installsitelib};'`, won't need changing everytime you bump the perl version